app/assets/stylesheets/layouts/starrylight.scss

```diff
@@ -1,86 +1,222 @@
 @import 'variables';
+
+/* - header */
+$bg_color_header_top: transparent; // top header strip for logo
+$bg_color_header_middle: #41416E; // middle header strip with user items
+$bg_color_header_bottom: #6F658D; // bottom header strip with navigation
+
 #header {
-  background-color: #6F658D;
+  background-color: $bg_color_header_bottom;
   background-image: url(image-path("layouts/starry.jpg"));
   background-position: right top;
 }
-#logo, #nav-bottom { background-color: transparent !important; }
-#user-info, #nav-top { background-color: #41416E; background-color: rgba(65, 65, 110, 0.9); }
-
-body { background-color: #B0AFB7; }
-
-th, .content-header { background-color: #42416E; }
-.sub { color: #F3F3F3; }
-.sub, .post-navheader, .flash.breadcrumbs { background-color: #8287A2; }
-.selector-popup, .single-description, .subber, .form-table-ender { background-color: #6F658D; color: #DADADA; }
-th, .single-description, .content-header {
-  a { color: #F3F3F3; }
-  a:visited { color: #C6C6E5; }
+#logo { background-color: $bg_color_header_top !important; }
+#user-info, #nav-top { background-color: $bg_color_header_middle; background-color: rgba(65, 65, 110, 0.9); }
+
+/* - links */
+$font_color_link: #666284; // link
+$font_color_link_visited: #C6C6E5;
+
+a { color: $font_color_link; }
+a:visited { color: #484357; } //TODO: inconsistency between this a:visited and others, ???? this was previous at the bottom of the documment, was it overriding?
+
+/* - flashes */
+// these are the notifications that appear when you submit a form or get a message
+// TODO: See if we can differentiate these
+$bg_color_flash_error: #CDC7E7; // error
+$bg_color_flash_success: #CDC7E7; // success
+$bg_color_flash_inbox: #CDC7E7; // new message
+$bg_color_flash_breadcrumbs: #8689B7; // breadcrumbs ("Continuities » Continuity » Section » Thread")
+$font_color_breadcrumbs_bold: #F3F3F3;
+
+.flash.breadcrumbs {
+  background-color: $bg_color_flash_breadcrumbs;
+  &, a, a:visited { color: $font_color_link_visited; }
+  b { color: $font_color_breadcrumbs_bold; } // TODO: Investigate why this is differentiated
 }
+.flash.success { background-color: $bg_color_flash_success; }
+.flash.error { background-color: $bg_color_flash_error; }
+.flash.inbox { background-color: $bg_color_flash_inbox; }
 
-$EVEN_COLOR: #E3E3E3;
-$ODD_COLOR: #D8D8D8;
+/* - main page */
+$bg_color_page: #B0AFB7; // overall page
+$font_color_page: #DADADA;
 
-#post-menu-box {
-  border-top: 1px solid #808080;
-  border-top: 1px solid rgba(128,128,128,0.5);
-  background-color: #BCBCBC;
+body { background-color: $bg_color_page; color: $font_color_page; }
```

Comment:
`color` is unlikely to have an impact in starrylight but this was in fact added again – can you maybe comment it out until I return to Fix The Text Colors?